testing/chain.go

```diff
@@ -124,7 +125,17 @@ func NewTestChain(t *testing.T, coord *Coordinator, chainID string) *TestChain {
 
 // GetContext returns the current context for the application.
 func (chain *TestChain) GetContext() sdk.Context {
-	return chain.App.BaseApp.NewContext(false, chain.CurrentHeader)
+	return chain.App.GetBaseApp().NewContext(false, chain.CurrentHeader)
+}
+
+// GetSimApp returns the SimApp to allow usage ofnon-interface fields.
+// CONTRACT: This function should not be called by third parties implementing
+// their own SimApp.
+func (chain *TestChain) GetSimApp() *simapp.SimApp {
+	app, ok := chain.App.(*simapp.SimApp)
+	require.True(chain.t, ok)
+
+	return app
```

Comment:
Other app devs would need to add their own function to get access to their app. They could probably do this by wrapping our chain struct with their own chain which adds this one function